bin/sl-run.js

```diff
@@ -16,6 +16,14 @@ var config = require('../lib/config'); // May exit, depending on argv
 var log = config.logger;
 var tracer = require('../lib/tracer');
 
+if (process.env.STRONGLOOP_APM_ENABLE) {
+  try {
+    require('../lib/apm')();
+  } catch (err) {
+    log.warn('supervisor failed to load APM modules: %s', err.message);
```

Comment:
I would prefer the opposite, no scary warnings in the normal case, no APM, but log the fact that we loaded APM, and from what location, if we do.